main/integration-tests/__test__/api-tests/common/migration/my-study-migration.test.js

```diff
@@ -0,0 +1,95 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../../support/setup');
+const errorCode = require('../../../../support/utils/error-code');
+
+describe('Migration for My Studies scenarios', () => {
+  let setup;
+  let adminSession;
+  let defaultUser;
+  let defaultUserSession;
+
+  beforeAll(async () => {
+    setup = await runSetup();
+    adminSession = await setup.defaultAdminSession();
+    // Create cognito user to migrate ownership to
+    // Proper way that currently doesn't work:
+    // const username = setup.gen.username();
+    // defaultUser = adminSession.resources.users.defaults({ username });
+    // defaultUser = await await adminSession.resources.users.create(defaultUser);
+    // await adminSession.resources.users.user(defaultUser.uid);
+    // Workaround:
+    defaultUserSession = await setup.createResearcherSession();
```

Comment:
This is the right way to create users, follows the pattern we follow in almost all of our integration test suites (therefore, wouldn't call this a workaround either).

Reply:
Got it. I thought I was going to switch to creating the user after the user create method was fixed. So I left this here for when that was in place. I will stick to just using the session logic.